src/templates/base.html

```diff
@@ -54,6 +55,7 @@
     {% endblock %}
     {% block scripts %}
       <script src="/static/js/language.js"></script>
+      <script src="/static/js/contributors.js"></script>
```

Comment:
Move this out of the base template (we don't want to load contributor-specific JS on all pages).